blocklylib-vertical/src/main/java/com/google/blockly/android/ui/vertical/VerticalBlockViewFactory.java

```diff
@@ -133,6 +134,11 @@ protected FieldView buildFieldView(Field field) {
                 colorView.setWorkspaceHelper(mHelper);
                 break;
             }
+            case Field.TYPE_ANGLE: {
+                BasicFieldAngleView angleView = (BasicFieldAngleView) fieldView;
+                angleView.setWorkspaceHelper(mHelper);
+                break;
+            }
```

Comment:
are you talking about [this](https://github.com/google/blockly-android/blob/master/blocklylib-vertical/src/main/java/com/google/blockly/android/ui/vertical/VerticalBlockViewFactory.java#L142-L146) ?

"fieldView == null"   is false here
```
 if (fieldView == null) {
      fieldView = super.buildFieldView(field);
 }
```